tools/installer/requirements.txt

```diff
@@ -1 +1,2 @@
 mutagen==1.31
+ensurepip
```

Comment:
What about
* lxml required by lyricsmania
* BeautifulSoup required by lyricswiki
* grp, cairo required by daapserver
* cairo (again), required by some other plugins
* pyHook required by winmmkeys (only useful on Windows, obviously)

Ideally this list would be autogenerated to reduce maintenance and make errors less likely.